bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -157,8 +157,10 @@ private func dummy() {
     _ = NSLocalizedString("Featured Image", comment: "")
     _ = NSLocalizedString("File block settings", comment: "")
     _ = NSLocalizedString("File name", comment: "")
+    _ = NSLocalizedString("Font Size", comment: "")
     _ = NSLocalizedString("From clipboard", comment: "")
     _ = NSLocalizedString("Gallery caption. %s", comment: "translators: accessibility text. %s: gallery caption.")
+    _ = NSLocalizedString("Get support", comment: "")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/editor-help/index.native.js#L160:L160)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.